cli/src/main.rs

```diff
@@ -188,10 +188,11 @@ async fn main() {
             info!("{:?}", tip_accounts);
         }
         Commands::MempoolAccounts { accounts } => {
-            info!(
-                "waiting for mempool transactions that write-locks accounts: {:?}",
-                accounts
-            );
+            let accounts = accounts
+                .into_iter()
+                .map(|a| a.to_string())
+                .collect::<Vec<String>>();
```

Comment:
why have Pubkey input if just converting to string anyway?  why not just leave as string param?

Reply:
Main reason is to have type info earlier, and exit sooner if incorrect params are passed